tools/cpp/cc_toolchain_config.bzl

```diff
@@ -685,7 +698,7 @@ def _impl(ctx):
                 ),
             ],
         )
-    elif (ctx.attr.cpu == "freebsd"):
+    elif (ctx.attr.cpu == "freebsd" or ctx.attr.cpu == "openbsd"):
```

Comment:
nit: Break in two lines for consistency with the rest?

Reply:
Done.